### ISSUE TITLE
Multiple Profiles per account

### ISSUE THREAD

**Issue by ShakedGold** [Stremio/stremio-features#622]
Basically the title, one account can create multiple profiles with their own watch history and library but the same add-ons for all the profiles under the account, maybe with logos kind of like netflix

**Comment by NaeemChakera** [comment]
If it's not possible to have multiple profiles on one account, at least make it possible to login and logout easier without having to put your passwords so we can have every person's account easily accessible from one machine, something like how you log in to multiple accounts on Facebook or Instagram.

**Comment by scalpmonkey** [comment]
+1

**Comment by tacosjs** [comment]
+1 for this!

**Comment by HitWithSpader** [comment]
Better parental controls would be nice as well!

**Comment by SmashBoii** [comment]
+1

**Comment by bratkure** [comment]
Absolutely. This is probably the biggest and most important feature that is missing... Especially for people who live with kids/parents.... Continue watching is personal thing imo. My worry is that we won't even get the answer about this request..  is it possible? I think should be, but i would love to hear someone from dev team is this refused, impossible or on waiting list?

**Comment by SmashBoii** [comment]
> Absolutely. This is probably the biggest and most important feature that is missing... Especially for people who live with kids/parents.... Continue watching is personal thing imo. My worry is that we won't even get the answer about this request.. is it possible? I think should be, but i would love to hear someone from dev team is this refused, impossible or on waiting list?

Continue watching is already a feature. If you don't have it login into Trakt.

**Comment by bratkure** [comment]
> > Absolutely. This is probably the biggest and most important feature that is missing... Especially for people who live with kids/parents.... Continue watching is personal thing imo. My worry is that we won't even get the answer about this request.. is it possible? I think should be, but i would love to hear someone from dev team is this refused, impossible or on waiting list?
> 
> Continue watching is already a feature. If you don't have it login into Trakt.

You have missed a point completely... I don't want others that use my tv to see my continue watching list or add their own content on the list... So for tv app profiles should be a big deal.

**Comment by ShakedGold** [comment]
Any updates on this?
Id love to contribute unfortunately, I don't know rust

**Comment by roisec** [comment]
+1

**Comment by inverse** [comment]
This feels like a nice feature and would bring streamio to be more in line with other offerings for folks living with others/families. 

Would be great if someone from the core team could give their input on this as something they would accept. Don't like directly tagging but @TRtomasz would you be able to give some input?

@ShakedGold I saw they have a new issue template which probably we should adopt for this issue to be more prepped for this.

See: https://github.com/Stremio/stremio-features/blob/master/.github/ISSUE_TEMPLATE/feature_request.md?plain=1

**Comment by cerels** [comment]
I'm not a dev but, would it be easier if Instead of having multiple profiles per account, you could have multiple accounts in the same app?

**Comment by ShakedGold** [comment]
@inverse did it :)

**Comment by pdrgrl** [comment]
Please, please, please, this would be exactly what I need, and apparently many others. Hopefully it will be implemented soon! 🙏

**Comment by hrafnfloki** [comment]
Definitely should implement this feature.

**Comment by MullyCap2004** [comment]
> Definitely should implement this feature.

Yup

**Comment by strugler** [comment]
This is absolutley will be a game changer. I currently face the issue with kids and mrs. The watch list gets messed up because of that. Please please add profile options!

**Comment by goncaloaguer** [comment]
This would be super useful!

**Comment by omgitsyasir** [comment]
agreed, would love to have this

**Comment by tzouvarasc** [comment]
+1

**Comment by hvk23** [comment]
+1, just started using stremio and love it but this is the biggest barrier to wife and kids adopting usage

thanks for all of the devs work, truly amazing software!

**Comment by OfficialCritz** [comment]
+1
We (me and my family) would love this feature!

**Comment by SamuraiJr** [comment]
Not sure how this could take this long to implement, perheps it would be easier to just be able to choose different accounts instead of profiles. But having others in my household use my account is really annyoing in terms of Trakt, Watchlist, etc.

**Comment by hypist** [comment]
+1
I would absolutely love to have this feature. It would make the experience so much better for those of us with numerous people in the household.

I'm really hoping profiles get added.

**Comment by USFHunTer** [comment]
I hope this would be on the next update

**Comment by likewise000** [comment]
I absolutely need this!! I really hope such a feature could be added soon.

For me it's probably the *one* thing missing from Stremio right now.

**Comment by HitWithSpader** [comment]
Not a dev, but one way to possibly implement this is to have separate profiles with different libraries and continue watching under the same account and addons

**Comment by Epirr** [comment]
This is the one missing feature it sorely needs imo

The current workaround is to clone the APK but this isn’t possible for those with an Apple TV

**Comment by swift148** [comment]
.

**Comment by pedrogiroldo** [comment]
+1

**Comment by CrissZollo** [comment]
+1

**Comment by prasanaramanaa** [comment]
+1

**Comment by N0amLevi** [comment]
+1

**Comment by Mr-Nobody1** [comment]
A much needed feature. Would love for the dev team to comment on this.

**Comment by migugu00** [comment]
+1

**Comment by MadOokami** [comment]
+2

**Comment by AdamyanMax** [comment]
+1

**Comment by Fabiobatta** [comment]
+1

**Comment by ciurlaro** [comment]
I'm a dev and would like to help, but some guidance is definitely needed from the main Devs!

**Comment by bartned** [comment]
Would be nice!

**Comment by N0amLevi** [comment]
Yes

**Comment by Nrwh1** [comment]
+1

**Comment by Krule** [comment]
+1

**Comment by Slayer1935** [comment]
+1

**Comment by CrissZollo** [comment]
This feature has gotten some more traction. 

@kKaskak or @tymmesyde is this something that can be implemented as a component after the login? Storing logins for easier switching? Or does the code need to be fully rewritten for this to work?

**Comment by DRG2Boss** [comment]
Yes please!!

**Comment by AW1534** [comment]
+1 🥹

**Comment by gitomit** [comment]
Would love this! I've made a design how it could look on Stremio. 

<img width="1394" height="1045" alt="Image" src="https://github.com/user-attachments/assets/5f910177-6844-404e-ba47-7a5f293a5865" />
https://gyazo.com/947de7a4dbe8e445a46d81cb88c918e5

Maybe some features of the profiles:
- Only the "admin" can create or remove profiles
- Password protected profiles, as I maybe don't want everyone snooping around in my profile or messing up my watch list
- Auto login to last profile based per device

**Comment by gblass** [comment]
This is a must feature, need to create a Kids Account Profile on my LG TV.

**Comment by groundyy** [comment]
this should be a main focus

**Comment by justj0sef** [comment]
Multiple profiles or way to make sure my kids don't watch things they shouldn't.

**Comment by TecEash1** [comment]
+1

**Comment by giggga-chigga** [comment]
+1

**Comment by aitronz** [comment]
+1

**Comment by kKaskak** [comment]
done check https://stremio.com/donate user profiles feature

it includes: 
1. Up to 6 user profiles per account

<img width="1536" height="1040" alt="Image" src="https://github.com/user-attachments/assets/21777cf1-4a2f-4c4a-b67a-8bcb1d789e89" />

2. Optional PIN protection per profile
<img width="535" height="541" alt="Image" src="https://github.com/user-attachments/assets/84fe37f0-7c21-44f8-928a-a2f35d32736b" />

3. Allow / disable addon management per profile
<img width="796" height="134" alt="Image" src="https://github.com/user-attachments/assets/49ef98e1-c617-4f33-868a-40de4e5fa212" />

4. management per profile of addons and catalogs

<img width="1515" height="1036" alt="Image" src="https://github.com/user-attachments/assets/bb86ae79-a6bf-4806-b699-709d2d8f48ca" />

5. Kid profiles and kids only catalogs
<img width="1217" height="1008" alt="Image" src="https://github.com/user-attachments/assets/f8d27437-cbdb-40e6-a9c8-2e25b75dbdaf" />

6. Reorder catalogs, hide and rename as well


Works on all platforms except linux from y-day release
Windows macOS android mobile android tv LG TV Samsung TV and even new iOS app
